service/k8sclient/kubectl/kubectl.go

```diff
@@ -38,8 +38,9 @@ import (
 )
 
 const (
-	// FIXME: change to a real default kubectl.
-	defaultKubectl = "minikube kubectl --"
+	dbaasToolPath      = "/opt/dbaas-tools/bin"
+	defaultKubectl     = dbaasToolPath + "/kubectl-1.16"
+	minikubeDevKubectl = "minikube kubectl --"
```

Comment:
Isn't it https://github.com/percona-platform/dbaas-controller/pull/58 ?

Reply:
yes, it is, and I'll merge conflicts. I wasn't sure which one will be merged first.
This changes was required for my task as well.

Reply:
Then maybe we can just close the other one?

Reply:
No, I pulled changes from his PR and going to merge this PR after @askomorokhov PR.